src/plugins/ExecuteJob/ExecuteJob.Metadata.js

```diff
@@ -13,14 +13,27 @@ define([
         this.createdMetadataIds = {};
         this.subGraphs = {};
         this.plotLines = {};
+        this.images = {};
+    };
+
+    ExecuteJob.prototype.SEPARATORS = {
+        JOB_GRAPH: '/',
+        GRAPH_SUBGRAPH: '/',
+        SUBGRAPH_LINE: '/L',
+        SUBGRAPH_IMAGE: '/I'
+    };
+
+    ExecuteJob.prototype.createId = function (parentId, childRelID, sep) {
+        return parentId + sep +childRelID;
     };
 
     // TODO: Add tests
+    // TODO: Refactor this into three different functions
```

Comment:
Does this need to be addressed?

Reply:
Yeah, probably in a different issue/PR combo I think, either way, I Just added a note as a reminder.

Reply:
This TODO has been addressed in this PR.